SimTKcommon/Simulation/include/SimTKcommon/internal/StateImpl.h

```diff
@@ -158,50 +233,63 @@ class CacheEntryInfo {
     CacheEntryInfo& deepAssign(const CacheEntryInfo& src) {
         assert(src.isReasonable());
```

Comment:
`isReasonable()` is a private method defined right there in that internal class (CacheEntryInfo). Not for external consumption. It is a Debug-mode-only bug catcher.